libcontainer/nsenter/nsexec.c

```diff
@@ -713,10 +713,12 @@ void send_idmapsources(int sockfd, pid_t pid, char *idmap_src, int idmap_src_len
 		ret = sys_mount_setattr(fd_tree, "", AT_EMPTY_PATH, &attr, sizeof(attr));
 		if (ret < 0) {
 			sane_kill(pid, SIGKILL);
-			if (errno == EINVAL)
-				bail("failed to change mount attributes, maybe the filesystem doesn't supports ID-mapped mounts");
+			if (errno == ENOSYS)
+				bail("failed to use mount_setattr(2), the kernel doesn't supports ID-mapped mounts");
+			else if (errno == EINVAL)
+				bail("failed to use mount_setattr(2), maybe the filesystem doesn't supports ID-mapped mounts: %s", idmap_src);
```

Comment:
`maybe` means support or not? I suggest change to:
```
bail("failed to use mount_setattr(2), the filesystem doesn't support this ID-mapped mount: %s, error msg: Invalid argument", idmap_src);
```

Reply:
Changed to that, but removed the ": Invalud argument". That is added by the bail macro, it will be duplicated if I put it like that.